index.bs

```diff
@@ -184,10 +191,14 @@ Cancelling the socket's ReadableStream and closing the socket's WritableStream d
 
 <h3 id="methods">Methods</h3>
 
-<h4 id="close-method">close()</h4>
+<h4 id="close-method">close(optional any reason)</h4>
 
 The {{close()}} method closes the socket and its underlying connection. It returns the same promise as the {{closed}} attribute.
 
+When called, the {{ReadableStream}} and {{WritableStream}} associated with the {{Socket}} will
+be canceled and aborted, respectively. If the {{reason}} argument is specified, the {{reason}}
```

Comment:
```suggestion
be cancelled and aborted, respectively. If the {{reason}} argument is specified, the {{reason}}
```

Reply:
heh, FWIW, while both forms are considered to be correct, `canceled` with one `l` is far more common in American English.

Reply:
ahh, my bad :)